services/influxdb/service.go

```diff
@@ -187,13 +191,25 @@ type subInfo struct {
 
 func (s *influxdb) Open() error {
 	if !s.disableSubs {
-		return s.linkSubscriptions()
+		err := s.linkSubscriptions()
+		if s.subscriptionSyncInterval != time.Duration(0) {
+			s.subSyncTicker = time.NewTicker(s.subscriptionSyncInterval)
+			go func() {
+				for _ = range s.subSyncTicker.C {
+					s.linkSubscriptions()
+				}
+			}()
+		}
+		return err
 	}
 	return nil
 }
 
 func (s *influxdb) Close() error {
 	var lastErr error
+	if s.subscriptionSyncInterval != time.Duration(0) {
```

Comment:
I prefer `if s.subscriptionSyncInterval != 0` as there is less to read and as a result reads cleaner.